### DISCUSSION TITLE
Code block .. style

### DISCUSSION THREAD

**Discussion by Haqbani** [Akifyss/obsidian-border#284] [Q&A]
Hi everyone, how to change the code block to contain more colors for syntax.. are there any open source codes ? How to apply them ?

Thanks!

**Comment by Akifyss** [comment]
Hello, you can change the style of code blocks in this style settings section:
![image](https://github.com/Akifyss/obsidian-border/assets/23567154/94c101c5-abbc-4aec-ae2f-94382e3eb5c6)
